docs/get-started/cardano-serialization-lib/create-react-app.md

```diff
@@ -0,0 +1,300 @@
+---
+id: create-react-app
+sidebar_position: 2
```

Comment:
This is a great addition. Could you please change the sidebar_position so that this example goes to the end of the current "Serialization-Lib" documentation?

Reply:
We did that. :)) Open to better ways to address this if you want to share your thoughts.